documentation/modules/deploying/proc-exposing-kafka-bridge-service-local-machine.adoc

```diff
@@ -20,15 +20,15 @@ kubectl get pods -o name
 
 pod/kafka-consumer
 # ...
-pod/quickstart-bridge-589d78784d-9jcnr
-pod/strimzi-cluster-operator-76bcf9bc76-8dnfm
+pod/my-bridge-bridge-7cbd55496b-nclrt
+pod/strimzi-cluster-operator-<version>-5886bccd9c-4nsxc
```

Comment:
Hmm ... the `<version>` part will be there only when installed through OLM. It might be also in different namespace. Maybe we can just remove it? I do not think the operator pod is really important for the procedure, so we avoid any question about why is it different here or there. Sorry.

Reply:
All things considered, I think removing it makes the most sense!